content/blog/2025/12/29/2025-12-29-election-results-2025.adoc

```diff
@@ -0,0 +1,72 @@
+we congratulate all newly elected board members and officers!
+We also thank all candidates and voters who participated this year.
+
+Election results:
+
+* link:/blog/authors/strangelookingnerd[Daniel Krämer] joins link:/blog/authors/NotMyFault[Alexander Brandes], link:/blog/authors/jonesbusy[Valentin Delaye], link:/blog/authors/slide_o_mix-board[Alex Earl] and link:/blog/authors/basil/[Basil Crow] on the link:/project/governance/#governance-board[Jenkins Governance Board]
+* link:/blog/authors/stefan_spieker/[Stefan Spieker] will serve as link:/project/team-leads/#events[Events Officer] (uncontested)
+* link:/blog/authors/timja/[Tim Jacomb] will continue as link:/project/team-leads/#release[Release Officer] (uncontested)
+* link:/blog/authors/dduportal/[Damien Duportal] will continue as link:/project/team-leads/#infrastructure[Infrastructure Officer] (uncontested)
+* link:/blog/authors/wadeck/[Wadeck Follonier] will continue as link:/project/team-leads/#security[Security Officer] (uncontested)
+* link:/blog/authors/krisstern/[Kris Stern] will serve as link:/project/team-leads/#documentation[Documentation Officer]
+
+The board positions and officer roles are an essential part of the Jenkins project and we are excited to see contributors taking these roles.
+
+The board continues to require that no single company may have a majority on the Jenkins governance board.
+
+== Governance Board election details
+
+This year we had three candidates for the Jenkins Governance Board, two for the Documentation Officer, and one for Events Officer.
+All of them are awesome community leaders who actively contribute to the Jenkins project and represent its users.
+
+We used the https://civs1.civs.us/[Condorcet Internet Voting Service] that allows voters to rank their choices rather than just picking their one favorite choice.
+You can find voting results in the the link:TODO[documentation officer poll].
+
+All board members are elected for a 2-year term.
+The estimated end of the term for them is December 15, 2027.
+
+== Officer election details
+
+All 5 officer positions were up for election this year.
+These roles have a 1-year term, with the estimated end of term on December 15, 2026.
+After the initial review of nominations and confirmations with potential candidates, 3 officer positions were uncontested:
+
+* **Stefan Spieker** - link:/project/team-leads/#events[Events Officer]
+* **Damien Duportal** - link:/project/team-leads/#infrastructure[Infrastructure Officer]
+* **Wadeck Follonier** - link:/project/team-leads/#security[Security Officer]
+
+Thanks to all Jenkins officers for their continued leadership!
+Officers take responsibility for many day-to-day processes in the Jenkins community and lead the contributor teams working on them.
+It requires significant time commitment and it is not taken for granted.
+
+=== Statistics
+
+This year we had link:https://community.jenkins.io/g/election-voter-2025[66 registered voters] and around TODO actual votes.
```

Comment:
We are still missing the `TODO actual votes` here

Reply:
39 votes were cast.